backend/packages/Upgrade/src/api/middlewares/ErrorHandlerMiddleware.ts

```diff
@@ -97,6 +97,10 @@ export class ErrorHandlerMiddleware implements ExpressErrorMiddlewareInterface {
             message = error.message;
             type = SERVER_ERROR.QUERY_FAILED;
             break;
+          case 409:
+            message = error.message;
+            type = SERVER_ERROR.DUPLICATE_KEY;
```

Comment:
409 is a general status, duplicate "key" is pretty specific. i know this is the only planned usage of 409 but this is what I'm talking about avoiding making specific custom code to just handle one specific case. The type should be something more general right?

Reply:
This code should not be here. We don't want all 409 status codes to be treated as a `DUPLICATE_KEY` error.

Reply:
It seems like this code is still here? @RidhamShah Shouldn't we delete or update this code as @danoswaltCL or @VivekFitkariwala suggested?